recipes/dialign2/build.sh

```
@@ -5,11 +5,23 @@ mkdir -p ${PREFIX}/bin
+export LIBRARY_PATH="${PREFIX}/lib"
+export CFLAGS="${CFLAGS} -O3 -L${PREFIX}/lib"
+
+if [[ `uname` == "Darwin" ]]; then
+	export CFLAGS="${CFLAGS} -Wno-implicit-int -Wno-implicit-function-declaration"
+else
+	export CFLAGS="${CFLAGS}"
+fi
+
 cd src
 
 sed -i.bak "s|strcpy ( dialign_dir , \"DIALIGN2_DIR\" );|strcpy ( par_dir , \""${PREFIX}"/share/dialign2\" );|g" dialign.c
```

Shellcheck

[warning] The surrounding quotes actually unquote this. Remove or escape them.

(SC2027)